src/Logary.Facade/Facade.fs

```diff
@@ -259,31 +426,6 @@ module internal Formatting =
     let formatInstant (utcTicks : int64) =
       (DateTimeOffset(utcTicks, TimeSpan.Zero).ToString("o")) + ": "
 
-    let formatValue fields = function
```

Comment:
Hey it's an awesome piece of code, but regex went nuts when I tried to add format/alignment etc